src/style.css

```diff
@@ -1,3 +1,115 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector as it impacts performance. It's better to set styles only for elements that require them. If you need to reset the margin and padding, consider using a more specific selector or applying the reset to only the elements that need it.